Makefile

```diff
@@ -1,4 +1,5 @@
-VERSION = 0.88.1
+# This VERSION variable indicates the latest tag.
+VERSION = $(subst v,,$(shell git describe --abbrev=0 --tags))
```

Comment:
note: We guard to not to be able to ship added commits after tagged commit by our workflow (not GitHub Workflows).